### PR TITLE
Update kind to v0.8.1

### PR DESCRIPTION
**Pull request by fiunchinho**
## Checklist

- [ ] Make yourself familiar with following readme sections:
    - [ ] [Coding Standards](https://github.com/giantswarm/architect-orb#coding-guidelines).
    - [ ] [Development](https://github.com/giantswarm/architect-orb#development).
    - [ ] [Design and Goals](https://github.com/giantswarm/architect-orb#design-and-goals).
- [ ] :warning: Update changelog in [CHANGELOG.md](https://github.com/giantswarm/architect-orb/tree/master/CHANGELOG.md).
- [ ] :warning: After the release update architect orb version in [.circleci/config.yml](https://github.com/giantswarm/architect-orb/tree/master/.circleci/config.yml).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Fixed
+### Added
 
 - `integration-test-install-tools`: installs Helm 2.16.1.
```
Comment (kopiczko) [inline CHANGELOG.md:12]:
```suggestion
- `integration-test-install-tools`: install Helm 2.16.1.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Changed
+
+- `integration-test`: upgraded kind from `v0.7.0` to `v0.8.1`.
```
Comment (kopiczko) [inline CHANGELOG.md:16]:
```suggestion
- `integration-test`: upgrade kind from `v0.7.0` to `v0.8.1`.
```

**Comment by rossf7** [comment]
@fiunchinho Can we close this? Upgraded to kind 0.9.0 in #200

**Comment by fiunchinho** [comment]
Sure! Thanks

**Pull request closed without merging**